platform/engineering/frontend/platform-assignments.md

```diff
@@ -0,0 +1,40 @@
+# Overview 
+
+`vets-website` has several parts that require ongoing maintenance and improvement. These platform parts are assigned a primary and a secondary engineer. 
```

Comment:
```suggestion
The VSP frontend team is responsible for several parts that require ongoing maintenance and improvement. These platform parts are assigned a primary and a secondary engineer. 
```

Reply:
kind of a nit, but trying to distinguish between `vets-website` the repo and "FE-related things that VSP owns"